src/Microsoft.Azure.Relay/HybridConnectionListener.cs

```diff
@@ -994,6 +996,7 @@ public void AcceptSucceeded(HybridConnectionStream connection)
                     clientWebSocket.Options.SetBuffer(this.bufferSize, this.bufferSize);
                     clientWebSocket.Options.SetRequestHeader("Host", this.Address.Host);
                     clientWebSocket.Options.Proxy = this.listener.Proxy;
+                    clientWebSocket.Options.AddSubProtocol(this.subProtocol);
```

Comment:
Good questions:  Agreed on keeping current behavior if sender does not send any.  Ideally acceptor should be able to respond with the actual protocol chosen, out of the protocols sent by client.  Maybe an overload to the accept call with predicate to filter headers that should be returned to client side?  Anyway, these simple changes unblock me, since both sides do not care about the protocol field, it is just Winhttp that is unhappy.